### DISCUSSION TITLE
Is the package manager feature production ready?

### DISCUSSION THREAD

**Discussion by mrdulin** [oven-sh/bun#1992] [Q&A]
I want to improve the CI/CD speed of the React enterprise project. One of the works is to improve the speed of installing the dependencies.

Is it worth migrating npm to bun's package manager?

**Comment by chebread** [comment]
bun is really much faster than npm. It's well worth migrating. However, there are still a lot of minor bugs that bun like the ones I've asked in my current question (#1993).
